src/coreclr/jit/targetwasm.h

```diff
@@ -44,12 +44,12 @@
   #define FEATURE_EH_CALLFINALLY_THUNKS 1  // Generate call-to-finally code in "thunks" in the enclosing EH region, protected by "cloned finally" clauses.
   #define CSE_CONSTS               1       // Enable if we want to CSE constants
 
-  #define RBM_ALLFLOAT             RBM_F0
+  #define RBM_ALLFLOAT             REG_STK
```

Comment:
If we are deleting fake registers, it would be better to replace them with `REG_NA` instead of `REG_STK`.

`REG_STK` is the half-way point between real and fake registers itself :).

Reply:
Ok, reverted basically.  I added 
```c
  #define RBM_R0                   REG_NA
  #define RBM_F0                   REG_NA
  #define REG_F0                   REG_NA
```
Maybe I should just collapse the uses of `RBM/REG_F0` ?